### DISCUSSION TITLE
Windows 10 will not install it

### DISCUSSION THREAD

**Discussion by bekke99** [d2phap/ImageGlass#766] [General] [answered]
### System information:
 - Windows OS version:  10
 - ImageGlass version:  Newest

### To Reproduce
1. 
2. 
3. 
4. 

### Actual behavior:

Windows 10 will not allow the .exe to run - The blue box says:  Microsoft Defender Smartscreen blocked an unknown app to run...etc...App ImageGlass_7.6.4.30_x64.msi 
Virus total says its clean.

### Expected behavior:

Install on pc

**Comment by d2phap** [comment, marked as answer]
This is because the executable files are not signed.
You can click "More info" then Continue to open(https://github.com/d2phap/ImageGlass/discussions/764#discussioncomment-7873)
